python/tvm/rpc/tracker.py

```diff
@@ -348,7 +348,7 @@ def close(self, conn):
         if "key" in conn._info:
             for value in conn.put_values:
                 _, _, _, key = value
-                rpc_key = key.split(":")[0]
+                rpc_key = key.rsplit(":", 1)[0]
```

Comment:
Could you please add a test on this change to highlight what is the fix here? Probably `tests/python/unittest/test_runtime_rpc.py` is the right place for such test. Also, maybe it is better to create a separate PR with this fix.

Reply:
Moved into https://github.com/apache/tvm/pull/13420